<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Виртуальный журнал</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="addGroupButton">
        <property name="text">
         <string>Добавить группу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addStudentButton">
        <property name="text">
         <string>Добавить студента</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addSubjectButton">
        <property name="text">
         <string>Добавить предмет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="subjectsComboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="addGradeButton">
        <property name="text">
         <string>Добавить оценку</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" stretch="1,1,2">
      <item>
       <widget class="QListWidget" name="groupsList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="studentsList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="gradesTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <column>
         <property name="text">
          <string>Предмет</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Оценка</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>